### ISSUE TITLE
"This Video is Unavailable" message whenever I open a video

### ISSUE THREAD

**Issue by babylonianhina** [ftde0/yt2009#114]
Forgive me, I am a dumbass with very little web dev skills. I followed this [guide on reddit](https://new.reddit.com/r/oldyoutubelayout/comments/1e5enm6/ultimate_guide_on_how_to_host_a_yt2009_instance/) to make a YT2009 instance and for the past 2 days it was working great, but now every video I click on redirects me to the main page with the message "This Video is Unavailable". Is there any fix for this? Please be easy on me

**Comment by ftde0** [comment]
hi, are you running yt2009 on a vps? your ip range could be blocked by youtube which could prevent videos from loading.

**Comment by babylonianhina** [comment]
I was hosting it on a free Render server as per the tutorial I followed

**Comment by ftde0** [comment]
as mentioned to #117, this sounds like yt-related changes. an option to work around those is being worked on.

**Comment by busybox11** [comment]
Same issue here, getting this error on an Oracle Cloud Linux instance. I did have issues hosting Cobalt before, that seemed related to IP blocking. Wouldn't surprise me.

**Comment by babylonianhina** [comment]
Update: I don't get the warning and tutorial to sign in when I open a video, I just get the same error. Is there a way I can do this manually?

**Comment by ftde0** [comment]
you can manually run `node yt2009tvsignin.js` from the `/back/` directory. complete the setup, exit the script with `ctrl+c` and restart yt2009.

**Comment by babylonianhina** [comment]
I am so sorry, I am a novice and a dumbass so please forgive me if these constant questions are irritating, but can you give me a tutorial on how to go about that? I am not a coder or web dev by any means, just an enthusiast who followed a Reddit tutorial

**Comment by GachaBarry** [comment]
if you're hosting this on render, the onscreen instructions are in the logs

**Comment by babylonianhina** [comment]
> if you're hosting this on render, the onscreen instructions are in the logs

You are a lifesaver! thank you!

**Comment by ftde0** [comment]
great to hear it's solved! feel free to create a new issue should anything else arise

**Comment by yt2009fan2009** [comment]
same happens to me

**Comment by babylonianhina** [comment]
So I updated my Render instance to the latest fork of YT2009, and I get the video error again, except this time, the logs do not give any instructions.
It'll just say:
"your setup may require you to use
a youtube account to continue using
yt2009.

go to [http://127.0.0.1:80/gredir](http://127.0.0.1/gredir)
and return to this console window
for more information."

The link just brings me to a blank page

**Comment by SuperLuigiKillz04** [comment]
replace the domain name with your instance's name
